content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -66,48 +53,90 @@ When constructing your funnel, you can filter globally or on a step.
 
 Next, click a datapoint to **investigate the specific attributes** that might have affected conversion rates, such as page load speed, ease of navigation, or checkout experience.
 
+## Conversion computing metrics
 
-## Changing the visualization
+### How Datadog computes conversion metrics
+Consider a funnel with events `A → B → C` and event steps `A, A, A, B, C, C`. 
+
+
+In this case, Datadog counts one conversion. This is because the conversion calculation matches only the first occurrence of event **A** and the first occurrence of event **C** in the sequence. 
+
+
+To further illustrate, if the user performs the event sequence `A, A, A, B, C, C, A, B, C`, Datadog counts two conversions. The first conversion completes with the sequence `A, A, A, B, C, C`, and the second conversion begins with the next occurrence of `A`.
 
-{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_change_viz.mp4" alt="Click the visualization dropdown to select a different view" video=true >}}
 
+Datadog calculates the average time between steps based on the average duration between the first and last step of each conversion.
+
+If you analyze your funnel by **user** or by **Account**, you can define your conversion time frame in hours or days of the first event. The default time frame for conversions is one day (a 24-hour window, not a calendar date) to determine if a conversion happened or not.
+
+<div class="alert alert-info"> Any action or view that happens between two steps in a funnel does not impact the step-by-step or overall conversion rate. As long as step 1 and step 2 happen in the right order in a given session at least once, it counts as a single converted session.</div>
+
+### Choose a conversion counting method
+
+When computing your conversion, you can choose between two counting methods: **Unique** or **Total**. 
+
+- **Unique**: Counts conversion only once per session, user, or account. For example, if the user completes the funnel sequence `A → B → C` multiple times within the same session (for example, `A, B, C, A, B, C`), it counts as **one conversion**. The `Unique` setting counts only the first conversion per session (or per user, depending on your analysis scope).
+<br> 
+<br> 
+
+- **Total**: Counts a conversion each time the same session ID, user or account completes the defined funnel. Using the same example (`A, B, C, A, B, C`), this methods counts **two conversions**. The `Total` setting doesn't multiply conversions for repeated steps within a single flow, instead it counts complete flows, not the number of times an intermediate step is repeated. 
+
+To choose how conversions are counted in your funnel, beside **Conversion count**, select **Unique** or **Total**,
+
+{{< img src="product_analytics/journeys/funnel_analysis/funnel_analysis_conversion.png" alt="Select a conversion measure, whether Unique or Total, to determine how your session conversions are counted." style="width:80%;" >}}
+
+
+## Change the visualization
```

Comment:
You talk about visualization types earlier in the page, so it would be a good idea to link down to here in case a user isn't familiar with what they are!